org-code-javabuilder/validation/src/main/java/org/code/validation/PainterTracker.java

```diff
@@ -0,0 +1,37 @@
+package org.code.validation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PainterTracker {
```

Comment:
This feels to me like the `PainterLog` class, and the current `PainterLog` class more like ` PainterLogValidator` (esp if we moved all the getters here)?

Reply:
`PainterLog` is what will be used by levelbuilder validation code, while `PainterTracker` will be used to generate the log (which we can think of as an after the fact log, not a changing log). I'm going to comment these classes to make them clearer. I might also look into splitting into separate folders or something to make things super clear. That being said, we need to keep the getters in `PainterLog` so that levelbuilder validation code can access it.

Reply:
I ended up splitting out the support code into a sub-package